modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -2473,7 +2473,7 @@ describe('V2 Wallet:', function () {
           typedData.version = SignTypedDataVersion.V4;
           it('should sign typed data V4', async function () {
             typedData.txRequestId = txRequestId;
-            const signTypedDataTssSpy = sinon.spy(tssEthWallet, 'signTypedDataTss' as any);
+            sinon.spy(tssEthWallet, 'signTypedDataTss' as any);
```

Comment:
Is this used anymore in test?, If not kindly remove this

Reply:
I think you are looking at a stale version of the PR ([latest version](https://github.com/BitGo/BitGoJS/blob/a90f86e57c27e4e66bbaee705bfddd0b4d1c6e21/modules/bitgo/test/v2/unit/wallet.ts#L2476))